docs/pages/compiled/+Page.mdx

```diff
@@ -1,5 +1,5 @@
 import { UseBati, CommunityNote } from '../../components'
 
-<CommunityNote />
+<CommunityNote url="https://compiledcssinjs.com" />
 
 <UseBati>[Compiled](https://compiledcssinjs.com)</UseBati>
```

Comment:
```suggestion
<UseBati>Compiled</UseBati>
```

It's a bit confusing to repeat the link in the next sentence. Let's avoid this everywhere.